badger/cmd/flatten.go

```diff
@@ -43,17 +44,26 @@ func init() {
 	flattenCmd.Flags().IntVarP(&numVersions, "num_versions", "", 1,
 		"Option to configure the maximum number of versions per key. "+
 			"Values <= 0 will be considered to have the max number of versions.")
+	flattenCmd.Flags().StringVar(&keyPath, "encryption-key-file", "",
+		"Path of the encryption key file.")
 }
 
 func flatten(cmd *cobra.Command, args []string) error {
 	if numVersions <= 0 {
 		// Keep all versions.
 		numVersions = math.MaxInt32
 	}
+	encKey, err := getKey(keyPath)
+	if err != nil {
+		return err
+	}
 	opt := badger.DefaultOptions(sstDir).
 		WithValueDir(vlogDir).
 		WithNumVersionsToKeep(numVersions).
-		WithNumCompactors(0)
+		WithNumCompactors(0).
+		WithBlockCacheSize(100 << 20).
+		WithIndexCacheSize(200 << 20).
```

Comment:
Do we need a flat for this?

Reply:
I don't think so. We can set some static value here as it is now.